Core/src/org/sleuthkit/autopsy/report/ReportProgressPanel.java

```diff
@@ -44,6 +47,12 @@
         ERROR
     }
 
+    public enum Events {
+
+        COMPLETED,
+        NOT_COMPLETED
```

Comment:
NOT_COMPLETED is kind of a weird event name...  what about STARTED> or IN_PROGRESS?